benchmarks/transport_montecarlo_numba_formal_integral_p.py

```diff
@@ -8,36 +8,62 @@
 import tardis.transport.montecarlo.formal_integral as formal_integral
 from benchmarks.benchmark_base import BenchmarkBase
 from tardis import constants as c
+from tardis import run_tardis
+from tardis.io.configuration.config_reader import Configuration
 from tardis.model.geometry.radial1d import NumbaRadial1DGeometry
-from tardis.util.base import intensity_black_body
-
 
+# Error in all functions: Terminating: fork() called from a process already using GNU OpenMP, this is unsafe. 
+@skip_benchmark
 class BenchmarkMontecarloMontecarloNumbaNumbaFormalIntegral(BenchmarkBase):
     """
     Class to benchmark the numba formal integral function.
     """
 
+    def __init__(self):
+        super().__init__()
+        self.config = None
+        filename = "data/tardis_configv1_benchmark.yml"
+        path = self.get_relative_path(filename)
+        self.config = Configuration.from_yaml(path)
+        self.Simulation = run_tardis(
+            self.config, log_level="ERROR", show_progress_bars=False
+        )
+
+        self.FormalIntegrator = formal_integral.FormalIntegrator(
+            self.Simulation.simulation_state, self.Simulation.plasma, self.Simulation.transport
+        )
+
     @parameterize(
         {
-            "nu": [1e14, 0, 1],
-            "temperature": [1e4, 1, 1],
+            "Parameters": [
+                {
+                    "nu": 1e14,
+                    "temperature": 1e4,
+                },
+                {
+                    "nu": 0,
+                    "temperature": 1,
+                },
+                {
+                    "nu": 1,
+                    "temperature": 1,
+                }
+            ]
         }
     )
-    def time_intensity_black_body(self, nu, temperature):
+    def time_intensity_black_body(self, parameters):
+        nu = parameters["nu"]
+        temperature = parameters["temperature"]
         func = formal_integral.intensity_black_body
-        actual = func(nu, temperature)
-        print(actual, type(actual))
-        intensity_black_body(nu, temperature)
+        func(nu, temperature)
```

Comment:
I don't think we should reassign the function to `func`. Just call the function as-is